app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,14 @@
+class CommentsController < ApplicationController
+  def new
+    @new_comment = Comment.new
+  end
+
+  def create
+    post = Post.find(params[:post_id])
+    all_comments = params[:comment]
+
+    Comment.create(post: post, author: current_user, text: all_comments['comment'])
+
+    redirect_to user_posts_path, notice: 'Successfully created post'
+  end
```

Comment:
- [ ] Kindly consider fixing this issue when I create a comment the comments appear as an empty in the view like the following image
>![image](https://user-images.githubusercontent.com/53402600/206879011-e09acdaa-887f-4307-998f-67559de74007.png)